src/Approvals/SFA.DAS.Approvals.Api/Controllers/OverlappingTrainingDateRequestController.cs

```diff
@@ -59,5 +59,17 @@ public async Task<IActionResult> ValidateUlnOverlapOnStartDate(long providerId,
 
             return Ok(result);
         }
+
+        [HttpGet]
+        [Route("{apprenticeshipId}/getOverlapRequest")]
+        public async Task<IActionResult> GetOverlapRequest(long apprenticeshipId)
```

Comment:
the variable name in the route should be draftApprenticeshipId

Reply:
updated